src/plugins/timeConductor/Conductor.vue

```diff
@@ -175,29 +194,64 @@ export default {
                 start: timeFormatter.format(bounds.start),
                 end: timeFormatter.format(bounds.end)
             },
-            rawBounds: {
+            viewBounds: {
                 start: bounds.start,
                 end: bounds.end
             },
             isFixed: this.openmct.time.clock() === undefined,
             isUTCBased: timeSystem.isUTCBased,
-            showDatePicker: false
+            showDatePicker: false,
+            altPressed: false,
+            isPanning: false,
+            isZooming: false
         }
     },
+    created() {
+        document.addEventListener('keydown', (e) => {
+            if (e.key === 'Alt') {
+                this.altPressed = true;
+            }
+        });
+        document.addEventListener('keyup', (e) => {
```

Comment:
lets clean up the event listeners on destroy. I know that the time conductor is only going to be created and destroyed once with the life of the application, but its good practice, and promotes reusability of this component in the future.

Reply:
ah yes. thanks. done.